lib/datadog/ai_guard/configuration/settings.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require_relative "ext"
+
+module Datadog
+  module AIGuard
+    module Configuration
+      # AI Guard specific settings
+      module Settings
+        def self.extended(base)
+          base = base.singleton_class unless base.is_a?(Class)
+          add_settings!(base)
+        end
+
+        def self.add_settings!(base)
+          base.class_eval do
+            # AI Guard specific configurations.
+            # @public_api
+            settings :ai_guard do
+              # Enable AI Guard.
+              #
+              # You can use this option to skip calls to AI Guard API without having to remove library as a whole.
+              #
+              # @default `DD_AI_GUARD_ENABLED`, otherwise `false`
+              # @return [Boolean]
+              option :enabled do |o|
+                o.type :bool
+                o.env Ext::ENV_AI_GUARD_ENABLED
+                o.default false
+              end
+
+              # AI Guard API endpoint path.
+              #
+              # @default `DD_AI_GUARD_ENDPOINT`, otherwise /api/v2/ai-guard
+              # @return [String, nil]
+              option :endpoint do |o|
+                o.type :string, nilable: true
+                o.env Ext::ENV_AI_GUARD_ENDPOINT
+                o.default "/api/v2/ai-guard"
```

Comment:
This is not valid, either the customer defines the full URL with the environment variable:
```
DD_AI_GUARD_ENDPOINT=https://app.datadoghq.com/api/v2/ai-guard
```

Or we derive it using the `DD_SITE`:
```
'datad0g.com' -> 'https://app.datad0g.com/api/v2/ai-guard' 
'datadoghq.com' -> 'https://app.datadoghq.com/api/v2/ai-guard' 
'ap1.datadoghq.com' -> 'https://ap1.datadoghq.com/api/v2/ai-guard' 
...
```